documentation/usage/installation.md

```diff
@@ -2,18 +2,153 @@
+
+For this step we'll first give OpenGOAL the location of the `.iso` file we created earlier. Click the `SETUP` button in the launcher window to begin with.
+
+![Setup OpenGOAL with your .iso file](/docs/installation/opengoal_launcher_jd1_setup.png)
+
+This will open a window, and you'll need to navigate to the `.iso` file you created earlier in the setup here to continue. Once you've chosen the `.iso` file, the launcher will begin extracting the assets for the game.
+
+![Launcher extracting assets and completing compilation](/docs/installation/opengoal_launcher_jd1_compiling.png)
+
+This can take some time, but if you find it taking more than a few minutes try opening the `INSTALLATION LOGS` view below the progress bar, and see if there is still progress being made. If you don't see any changes here after a minute or two more, try closing and re-running the launcher.
+
+This process will complete loading the assets from your `.iso` file, and once it's done you'll be ready to run the game!
+
+![Starting Jak and Daxter in OpenGOAL](/docs/installation/opengoal_launcher_jd1_ready_to_start.png)
+
+Once you've reached this screen, you're all set up and ready to go! Just hit `PLAY` to start the game. The other controls are not something you'll need to play the game, as most of them are tools for developers, but do take note of the `SETTINGS AND SAVES` button if you want a link to your saves to move them or back them up, and of course the `UNINSTALL` button if you choose to do so down the line. The OpenGOAL launcher can also be uninstalled from the OpenGOAL installer used earlier in the instructions.
+
+That's everything! At this point you should have a working copy of the game to play. Join us in the [discord](https://discord.gg/VZbXMHXzWv) or finding streams/streaming yourself on [Twitch](https://www.twitch.tv/directory/game/Jak%20and%20Daxter:%20The%20Precursor%20Legacy) to get involved in the community.
+
+**ENJOY!**
 
 ## Linux
 
-TODO
+The instructions for Linux users are a bit more sparse than the Windows users, but most of the steps are identical.
+
+To begin with you'll first need to create a `.iso` file from your copy of Jak and Daxter, reading the dics for its content. The [Windows ISO instructions](installation#creating-your-iso) have some example instructions using a Windows program called ImgBurn, which you'll need an alternative to, but the rest of the instructions are the same.
+
+The biggest change is how you run OpenGOAL, as we don't provide a pre-built Linux installer, so the instructions for building the game are below.
+
+### Running OpenGOAL on Linux
+
```

Comment:
most of the questions for linux support tend to involve the steamdeck, as these are linux users that aren't used to linux.  It might be worth adding a steamdeck section, in which case this is a pretty good video tutorial that i believe is still accurate - https://youtu.be/Cv7mlCSHKxc?t=564

Reply:
Yeah wow that's a great video, for the intro to the project as well as the steamdeck setup. I think the only difference in the setup would be that the other discs should now be supported which I'll include as a text note with the link, but the rest still all seems good to go!